addon/locations/router-scroll.js

```diff
@@ -1,25 +1,27 @@
 import Ember from 'ember';
 
+const uuid = () => {
+  'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
```

Comment:
this implementation of the uuid function doesn't work. It needs to `return` the result of this `replace` function. Otherwise it will always be `undefined` and the `scrollMap` will not be populated